CHANGELOG.md

```diff
@@ -14,3 +14,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Integration with libFuzzer's `FuzzedDataProvider`.
 - Examples with tests.
 - Documentation with usecases, API etc.
+- Support for command line arguments for libfuzzer.
+- Environment variable to disable parsing of command line arguments for libfuzzer - `LUZER_NOT_USE_CLI_ARGS_FOR_LF`.
```

Comment:
Add tests that cover in different combinations:
- arguments specified Lua table
- arguments specified in console
- using `LUZER_NOT_USE_CLI_ARGS_FOR_LF`
- path to a seed corpus specified in Lua table and in console (Hint: libfuzzer prints a message `A corpus is not provided, starting from an empty corpus` when seed corpus is not specified and something like `INFO:    45167 files found in ../../corpus/torture/` when seed corpus is specified)